### ISSUE TITLE
Accessibility Insights: listview - <object> elements must have alternate text

### ISSUE THREAD

**Issue by RomanVakhterov** [winjs/winjs#1700]
Title: WCAG 1.1.1: <object> elements must have alternate text (object)
Tags: Accessibility, WCAG 1.1.1, object-alt​
​
Issue: <object> elements must have alternate text (object-alt - https://dequeuniversity.com/rules/axe/3.3/object-alt?application=msftAI)​

Element path: object​
​
Snippet: `<object style="display: block;position:absolute;top: 0;left: 0;height: 100%;width: 100%;overflow: hidden;pointer-events: none;z-index: -1;" data="about:blank" type="text/html" class="win-resizeinstrument win-disposable" tabindex="-1"></object>​`
​
How to fix: ​
Fix any of the following:​
  Element does not have text that is visible to screen readers​
  aria-label attribute does not exist or is empty​
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty​
  Element has no title attribute or the title attribute is empty​
  Element's default semantics were not overridden with role="presentation"​
  Element's default semantics were not overridden with role="none"​
​
